node/full.go

```diff
@@ -395,26 +395,37 @@ func (n *FullNode) Run(ctx context.Context) error {
 		return fmt.Errorf("error while starting data sync service: %w", err)
 	}
 
+	errCh := make(chan error, 1)
```

Comment:
please add a comment why size 1 makes sense here. I see 2 producers and 1 consumer so buffer of 1 makes sense to not block the producers as the consumer reads only the first err